docs/docs/changelog/v9-1.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: "v9-1-release"
+title: "v9.1"
+authors: ecalc-team
+tags: [release, eCalc]
+sidebar_position: -37
+---
+
+# eCalc v9.1
+
+## Features
+
+* Upgrade NeqSim to v3.0.5 (from v2.5.21). This can affect fluid/EoS operations, but we have not registered any changes. There may be a small speed up in fluid operations.
+* `CONTROL_MARGIN` is no longer experimental! It is also required for single speed and variable speed compressors.
+
+## Bug Fixes
+
+* Max Rate algorithm has been changed. The new version is a correct, but brute force algorithm. This will affect processing time negatively, but guarantee correctness.
+
+## Breaking changes
+
+* `CONTROL_MARGIN` is required for single speed and variable speed compressors.
```

Comment:
This was already mentioned as breaking change in v9.0. Why here?

Reply:
appearantly because it didnt happen in 9.0? check commits, it seemed to have been introduced in 9.1.. maybe there was another fix then?

Reply:
It is included in the changelog (autogenerated) for v9.0, that is not correct? https://github.com/equinor/ecalc/releases

Reply:
@frodehk you are right, i must have blinked :)

Reply:
fixed